homeassistant/components/switch/command_line.py

```diff
@@ -45,11 +48,12 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
         switches.append(
             CommandSwitch(
                 hass,
+                device_name,
```

Comment:
Rename this variable to `object_id`, as that is what it is going to be used for.